unzip/README.md

```diff
@@ -0,0 +1,57 @@
+# unzip command
+
+This is a simple `unzip` command.
+It's meant to be used with the binary-compatibility mode of Unikraft.
+
+The contents are:
+
+* /usr/bin/unzip: pre-built Linux ELF
```

Comment:
```suggestion
* `/usr/bin/unzip`: pre-built Linux ELF
```

Reply:
I see! I removed the hosts file. Thanks.